styleguide-app/Examples/TextInput.elm

```diff
@@ -60,89 +60,106 @@ example =
                 [ Control.view UpdateControl state.control
                     |> Html.fromUnstyled
                 , Heading.h3 [] [ text "TextInput.view { type_ = TextInput.text }" ]
```

Comment:
Oh! One other thing -- could we update these headings? They're funky now given the glorious new API

Reply:
Oh, you mean there's a new Heading API to switch to?  I'll give it a try.

Reply:
Oh! I meant "TextInput.view { type_ = TextInput.text }" was no longer quite correct

Reply:
Not a big deal though

Reply:
Hmm, maybe you meant just that the spacing is messy because headers have no built-in margins?   So ...  Grid layout!  
<img width="570" alt="Screen Shot 2020-04-14 at 10 56 15 AM" src="https://user-images.githubusercontent.com/1222/79257527-9882c300-7e3e-11ea-91e1-597eb974029d.png">